widgets/Menu.php

```diff
@@ -17,6 +17,39 @@ class Menu extends \yii\widgets\Menu
     public $submenuTemplate = "\n<ul class='treeview-menu' {show}>\n{items}\n</ul>\n";
     public $activateParents = true;
     public $defaultIconHtml = '<i class="fa fa-circle-o"></i> ';
+    private $noDefaultAction;
+    private $noDefaultRoute;
```

Comment:
It can be `false` or `string` of default route/action for current module/controller.
Don't like named.